daemon/algod/api/server/v2/handlers.go

```diff
@@ -51,8 +51,14 @@ import (
 	"github.com/algorand/go-codec/codec"
 )
 
-const maxTealSourceBytes = 1e5
-const maxTealDryrunBytes = 1e5
+// max compiled teal program is currently 8k
+// but we allow for comments, spacing, and repeated consts
+// in the source teal, allow up to 200kb
+const maxTealSourceBytes = 2e5
```

Comment:
Now that know `_` is allowed in int constants in Go, I would prefer `200_000` here